apps/web/src/scenes/GalleryPage/GalleriesPage.tsx

```diff
@@ -207,44 +208,46 @@ export default function GalleriesPage({ queryRef }: Props) {
         {query.userByUsername && (
           <UserGalleryHeader queryRef={query} userRef={query.userByUsername} />
         )}
-        <DndContext
-          sensors={sensors}
-          collisionDetection={closestCenter}
-          measuring={layoutMeasuring}
-          onDragStart={handleDragStart}
-          onDragEnd={handleDragEnd}
-          onDragOver={handleDragOver}
-          onDragCancel={() => setActiveId(null)}
-        >
-          <SortableContext items={sortedGalleryIds} strategy={rectSortingStrategy}>
-            <GalleryWrapper>
-              {sortedGalleries.map((gallery) => {
-                return (
+        <MobileSpacingContainer>
```

Comment:
only change here is wrapping everything in <MobileSpacingContainer/>

I added <MobileSpacingContainer/> to all these gallery content areas because we originally had a margin defined around the whole page. however the updated design is that the tab component ('Featured', 'Posts', 'Bookmarks', etc) takes up the full view width on mobile. because the tab component sits between the header and the rest of the page content, I had to remove the full page margin and add it specifically to the sections before and after the tav component.